torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6443,6 +6464,18 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            sample_inputs_func=partial(sample_inputs_softmax_variant, with_dtype=True),
            assert_autodiffed=True,
            supports_out=False),
+    OpInfo('nn.functional.normalize',
+           aten_name='normalize',
+           sample_inputs_func=sample_inputs_normalize,
+           dtypesIfCPU=floating_and_complex_types_and(torch.bfloat16),
+           dtypesIfCUDA=floating_and_complex_types_and(torch.half, torch.bfloat16),
+           skips=(
+               # RuntimeError: aliasOp != torch::jit::getOperatorAliasMap().end()
+               # INTERNAL ASSERT FAILED at "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":159,
+               # please report a bug to PyTorch.
+               SkipInfo('TestJit', 'test_variant_consistency_jit',),
```

Comment:
cc @eellison

Reply:
From what I understood so far, it's probably because `normalize` calls `norm` and `torch.div` (or `/` operator) in its definition: https://github.com/pytorch/pytorch/blob/88af4d84415855e0863a7138f87f96c203400efe/torch/nn/functional.py#L4437-L4441

Hence the graph created from: https://github.com/pytorch/pytorch/blob/88af4d84415855e0863a7138f87f96c203400efe/torch/testing/_internal/jit_metaprogramming_utils.py#L488-L491 doesn't consist of `aten::normalize` but instead has the nodes for `norm` and `div`. This leads to continuing from the loop here (no `if` condition satisfied): https://github.com/pytorch/pytorch/blob/88af4d84415855e0863a7138f87f96c203400efe/torch/csrc/jit/passes/utils/check_alias_annotation.cpp#L151-L153

And hence the final failure here: https://github.com/pytorch/pytorch/blob/88af4d84415855e0863a7138f87f96c203400efe/torch/csrc/jit/passes/utils/check_alias_annotation.cpp#L159

Was just trying to see what's happening. :) In case it helps!